src/virtual-machine.js

```diff
@@ -295,7 +296,27 @@ class VirtualMachine extends EventEmitter {
                 if (error) return reject(error);
                 resolve(res);
             });
-        });
+        })
+            .catch(error => {
+                try {
+                    const sb1 = new SB1File(input);
+                    const json = sb1.json;
+                    json.projectVersion = 2;
+                    return [json, sb1.zip];
```

Comment:
I assume this needs to be a Promise.resolve...?

```suggestion
                    return Promise.resolve([json, sb1.zip]);
```